app/models.py

```diff
@@ -11,3 +11,11 @@ class ChatResponse(BaseModel):
 
 class LearnRequest(BaseModel):
     text: str = Field(description="The text to be learned by the chatbot")
+
+#Practice
```

Comment:
`nit` top-level classes or functions should be surrounded by 2 blank lines
https://peps.python.org/pep-0008/#blank-lines

I have no idea why `ruff` does not raise the formatting error, but you might want to install `flake8` and use it as a linter for now.